.travis.yml

```diff
@@ -1,4 +1,25 @@
 language: node_js
 node_js:
   - 8.4.0
-script: "npm run lint && npm test"
+
+script:
+  - "npm run lint"
```

Comment:
do these need to be in quotes ? / havent seen it like this b4

Reply:
No, it is optional for yaml syntax, but I think it clarifies things sometimes.

Reply:
gotcha 👍